### PR TITLE
i.in.spotvgt.py: fixed E722 do not use bare 'except' warning

### PR DESCRIPTION
**Pull request by ShubhamSwati**
I have fixed the bare except warning in the file i.in.spotvgt.py

### CODE REVIEW

**Review by lucadelu — APPROVED** [review]

**Comment by ShubhamSwati** [comment]
I made change in r.unpack.py to fix bare except warning, kindly review.

**Comment by ShubhamSwati** [comment]
I made change in another file r.unpack.py to fix bare except warning, kindly review.

File: scripts/.flake8 [hunk 1]
```diff
@@ -19,7 +19,7 @@ per-file-ignores =
     v.report/v.report.py: F841
     db.out.ogr/db.out.ogr.py: F841
     g.extension/g.extension.py: F841, E722
-    r.unpack/r.unpack.py: E722
+
```
Comment (wenzeslaus) [inline scripts/.flake8:22]: No need for an empty line. Kind of "removing entry, removing line" rule.

File: scripts/.flake8 [hunk 2]
```diff
@@ -34,7 +34,7 @@ per-file-ignores =
     v.in.wfs/v.in.wfs.py: E722
     r.plane/r.plane.py: E722
     v.what.strds/v.what.strds.py: E722
-    i.in.spotvgt/i.in.spotvgt.py: E722
+
```
Comment (wenzeslaus) [inline scripts/.flake8:37]: Same as above. Extra line.

File: scripts/i.in.spotvgt/i.in.spotvgt.py [hunk 3]
```diff
@@ -155,7 +155,8 @@ def main():
         s = p.communicate()[0]
         if s == "application/x-zip":
             gscript.fatal(_("Please extract %s before import.") % infile)
-    except:
+    # code changed here, Exception included instead of bare except
```
Comment (wenzeslaus) [inline scripts/i.in.spotvgt/i.in.spotvgt.py:158]: No need for the text. This should be in the commit message. Extend PR description if needed. That's what we can use when writing the commit message when merging the PR.

File: scripts/r.unpack/r.unpack.py [hunk 4]
```diff
@@ -73,10 +73,9 @@ def main():
     os.chdir(tmp_dir)
     tar = tarfile.TarFile.open(name=input_base, mode="r")
     try:
-        data_names = [
-            tarinfo.name for tarinfo in tar.getmembers() if "/" not in tarinfo.name
-        ]
-    except:
+        data_names = [tarinfo.name for tarinfo in tar.getmembers() if "/" not in tarinfo.name]
```
Comment (wenzeslaus) [inline scripts/r.unpack/r.unpack.py:76]: Maybe a different black settings, but this line is too long and is thus not Black conformant. You can see it in the Black check. In any case, it is not really being modified by this PR, so even the white space should not change.

File: scripts/r.unpack/r.unpack.py [hunk 5]
```diff
@@ -73,10 +73,9 @@ def main():
...
+        # code changed here, Exception included instead of bare except
```
Comment (wenzeslaus) [inline scripts/r.unpack/r.unpack.py:77]: Same as above. This info goes to commit messages, not code, but I appreciate the effort to comment on the code.

**Review by wenzeslaus — CHANGES_REQUESTED** [review]
This needs just a little cleanup, otherwise it is good to go.

**Comment by ninsbl** [comment]
Bumping up milestone as 8.0.1 is due in two days, while this has not been part of RC1 and there has not been activity for some time.

**Review by wenzeslaus — APPROVED** [review]

**Comment by wenzeslaus** [comment]
I fixed the issues, so this can be merged.

**Pull request merged**